data/remotes.d/lvfs.conf

```diff
@@ -6,7 +6,8 @@ Title=Linux Vendor Firmware Service
 MetadataURI=https://cdn.fwupd.org/downloads/firmware.xml.@compression@
 ReportURI=https://fwupd.org/lvfs/firmware/report
 SecurityReportURI=https://fwupd.org/lvfs/hsireports/upload
-OrderBefore=fwupd
+OrderBefore=dell-esrt
+OrderAfter=backup,vendor,vendor-directory
```

Comment:
Isn't vendor-directory just an example?  I think this makes it a lot more confusing for someone who is using it as a _template_ and drops a new file in.  They would have no idea that they need to replace it in these other remote conffiles.

Reply:
> Isn't vendor-directory just an example

Google is using it "as-in" on ChromeOS :)

> and drops a new file in

I don't know if it makes more sense to edit the "worse" file to the "better" one. I guess whatever policy you have for download v.s. local remotes is probably going to be the same, i.e. lvfs probably wants the same dep order as say dell-embargo -- and vendor-directory-another-one wants the same dep order as vendor-directory.

Reply:
Maybe this is a fwupd.conf thing? i.e. `DuplicateReleasePolicy = local` for the common case?

Reply:
That seems reasonable.

Reply:
Okay, I've got a prototype that works -- adding ReleaseDedupe and ReleasePriority as different conf keys -- as they're different things and trying to squash them into one thing gets messy. I'll split it into a proper patch series tomorrow.

Reply:
Done; it's much better IMHO.